apps/codeforafrica/src/payload/collections/Members.js

```diff
@@ -0,0 +1,83 @@
+import { allCountries } from "../../lib/data/json/countries";
+import image from "../fields/image";
+import richText from "../fields/richText";
+import slug from "../fields/slug";
+import socialLinks from "../fields/socialLinks";
+import nestCollectionUnderPage from "../utils/nestCollectionUnderPage";
+
+const Members = {
+  slug: "members",
+  labels: {
+    singular: {
+      en: "Member",
+    },
+    plural: {
+      en: "Members",
+    },
+  },
+  access: {
+    read: () => true,
+    create: () => true,
+    update: () => true,
+  },
+  admin: {
+    useAsTitle: "name",
+  },
+  fields: [
+    image({
+      overrides: {
+        required: true,
+      },
+    }),
+    {
+      name: "name",
+      label: {
+        en: "Name",
+      },
+      type: "text",
+      required: true,
+    },
+    {
+      name: "title",
+      label: {
+        en: "Title",
+      },
+      type: "text",
+      required: true,
+    },
+    slug({ fieldToUse: "name" }),
+    {
+      name: "country",
+      type: "select",
+      options: allCountries,
+      label: { en: "country" },
+    },
+    richText({
+      name: "description",
+      label: {
+        en: "Description",
+        fr: "La description",
+        pt: "Descrição",
+      },
+      localized: true,
+      required: true,
+    }),
+    socialLinks({
+      name: "connect",
+      label: "Social Media Links",
+      required: false,
+    }),
+    {
```

Comment:
Perhaps Teams should be in its own collection as it is re-used for multiple people